.github/workflows/docker-publish.yml

```
@@ -0,0 +1,93 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '19 16 * * *'
+  push:
+    branches: [ master ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  # Use docker.io for Docker Hub if empty
```

Esse comentário não tá muito bom pra quem tá vendo um github actions pela primeira vez, talvez alguma coisa assim pode ser melhor:

```suggestion
  # The register is the docker image repository where we plan on uploading our images
  # it defaults to `docker.io` if unset.
  #
  # We are using `ghcr.io` which is the Github package repository and it is particularly useful
  # because it saves us the trouble of authenticating this script with an external provider,
  # since the script is already being executed inside Github.
```